themes/classic/assets/css/custom.css

```diff
@@ -2,3 +2,22 @@
  * Custom code goes here.
  * A template should always ship with an empty custom.css
  */
+@media screen and (max-width: 767px) {
```

Comment:
Theme css code should be set in the appropriate scss file (inside the `_dev` folder). This file is for shop customizations only, as stated in the comment above.

Reply:
@eternoendless I know the place was wrong :) I just didn't know where to place it. Thanks, I will update my PR